lib/server.js

```diff
@@ -271,6 +287,18 @@ function findPort(serverPort, optionsPort) {
   return 8888;
 }
 
+function findHostname(serverHostname, optionsHostname) {
+  if (typeof serverHostname !== 'undefined') {
```

Comment:
It's probably better to use a simple truthiness check rather than checking for undefined. That way localhost will be used if somebody passes null.

Reply:
same with empty string